src/components/common/TmBalance.vue

```diff
@@ -350,8 +351,9 @@ export default {
       }
     }
   },
-  mounted() {
-    this.setPreferredCurrency()
+  mounted: async function() {
+    this.$store.dispatch(`loadLocalPreferences`)
```

Comment:
why does this need to happen in this component, isn't hat something more global?

Reply:
well, it is the only place where it is being used, but you are right. I will investigate